sdk/communication/azure-communication-chat/swagger/swagger.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "Azure Communication Chat Service",
```

Comment:
Why is there still a static swagger being checked in? Is it on github and we can just link to the URL?

Reply:
I'm not aware of any decisions to stop checking in the swagger.

Reply:
We use the url of the API specs (it is a separate repo). Amr can give you the link, but for now we can go ahead and merge.
On the next round (the last one/GA version) we need to point to that one though.